test/emissions/emission-controller.spec.ts

```diff
@@ -20,28 +20,53 @@ import {
 import { currentWeekEpoch, increaseTime, getTimestamp, increaseTimeTo, startWeek } from "@utils/time"
 
 const defaultConfig = {
-    A: -166000,
-    B: 180000,
-    C: -180000,
-    D: 166000,
+    A: -166000000000000,
```

Comment:
@dimsome  are you ok with the values updated ?

Reply:
This matches the spreadsheet so looks good